cockatrice/src/interface/widgets/tabs/api/archidekt/display/archidekt_api_response_deck_entry_display_widget.cpp

```diff
@@ -0,0 +1,238 @@
+    backgroundPlateWidget = new BackgroundPlateWidget(this);
+    backgroundPlateWidget->setFixedHeight(120); // Adjust height to fit all labels
+
+    QVBoxLayout *plateLayout = new QVBoxLayout(backgroundPlateWidget);
+
+    // Add labels to the plate layout
+    QLabel *ownerLabel = new QLabel(QString("Owner: %1").arg(response.getOwner().getName()));
+    plateLayout->addWidget(ownerLabel);
+
+    QLabel *createdAtLabel = new QLabel(QString("Created: %1").arg(timeAgo(response.getCreatedAt())));
+    plateLayout->addWidget(createdAtLabel);
+
+    QLabel *updatedAtLabel = new QLabel(QString("Updated: %1").arg(timeAgo(response.getUpdatedAt())));
+    plateLayout->addWidget(updatedAtLabel);
+
+    // Add the shared plate to the header layout
+    headerLayout->addWidget(backgroundPlateWidget);
+
+    layout->addLayout(headerLayout);
+}
+
+void ArchidektApiResponseDeckEntryDisplayWidget::mousePressEvent(QMouseEvent *event)
+{
+    QWidget::mousePressEvent(event);
+    actRequestNavigationToDeck();
+}
+
+#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
+void ArchidektApiResponseDeckEntryDisplayWidget::enterEvent(QEnterEvent *event)
+#else
+void ArchidektApiResponseDeckEntryDisplayWidget::enterEvent(QEvent *event)
+#endif
+{
+    QWidget::enterEvent(event);
+    backgroundPlateWidget->setFocused(true);
+}
+
+void ArchidektApiResponseDeckEntryDisplayWidget::leaveEvent(QEvent *event)
+{
+    QWidget::leaveEvent(event);
+    backgroundPlateWidget->setFocused(false);
+}
+
+void ArchidektApiResponseDeckEntryDisplayWidget::setScaleFactor(int scale)
+{
+    scaleFactor = scale;
+    updateScaledPreview();
+}
+
+void ArchidektApiResponseDeckEntryDisplayWidget::onPreviewImageLoadFinished(QNetworkReply *reply)
+{
+    // Check if this is our reply
+    void *owner = reply->property("deckWidget").value<void *>();
+    if (owner != this) {
+        return; // not our reply
+    }
+
+    // Check that the requested URL matches what we asked
+    QUrl requestedUrl = reply->property("requestedUrl").toUrl();
+    if (requestedUrl != imageUrl) {
+        reply->deleteLater();
+        return;
+    }
+
+    QPixmap loaded;
+
+    if (reply->error() != QNetworkReply::NoError || !loaded.loadFromData(reply->readAll())) {
+        CardPictureLoader::getCardBackLoadingFailedPixmap(loaded, QSize(400, 400));
+    }
+
+    originalPixmap = loaded;
+
+    // universal fallback/design aspect ratio
+    static constexpr float DESIGN_RATIO = 150.0f / 267.0f;
+
+    // Always scale preview widget to this ratio
+    previewWidget->setAspectRatio(DESIGN_RATIO);
+    previewWidget->setPreviewWidth(400);
+
+    // Initial scaling
+    updateScaledPreview();
+
+    reply->deleteLater();
+}
+
+void ArchidektApiResponseDeckEntryDisplayWidget::updateScaledPreview()
+{
+    if (originalPixmap.isNull()) {
+        return;
+    }
+
+    int baseWidth = 400;
+    int newWidth = baseWidth * scaleFactor / 100;
+    int newHeight = int(newWidth * (150.0 / 267.0));
```

Comment:
might make sense to break out DESIGN_RATIO above into a class var so we can reuse it